scripts/comparator/run.sh

```diff
@@ -73,3 +73,6 @@ $comparator -input=$QUERY_FILE \
 -e=$END \
 -comparator=$COMPARATOR_WRITE \
 -regressionDir=$REGRESSION_DIR
+
+# Run PromQL testdata tests
+go test -v -timeout 30s -tags=compatibility -count=1 github.com/m3db/m3/src/query/test/
```

Comment:
nit: add newline

I think drop the `timeout 30s` as well, think that may lead to flakiness eventually if these tests get more fleshed out; I think CI jobs have a 10m liveness delay, so it won't hang CI, but we can always put it up to `300s` or something?